File: php/arrays/operations_on_elements/shifting_elements.php
#!/usr/bin/php8.0
<?php

$array = [2, 4, 6];

print("Initial state:\n\n");
print_r($array); print("\n");

// array_shift

array_shift($array);

print("After shift by array_shift(\$array):\n\n");
print_r($array); print("\n");

File: ruby/arrays/shifting_elements.rb
#!/usr/bin/ruby2.7

array = [2, 4, 6, 8, 10]

puts("Initial state:\n\n")
puts(array.inspect, "\n")

array.shift()

puts("After shift by array.shift():\n\n")
puts(array.inspect, "\n")

array.shift(0)

puts("After shift by array.shift(0):\n\n")
puts(array.inspect, "\n")

array.shift(1)

puts("After shift by array.shift(1):\n\n")
puts(array.inspect, "\n")

array.shift(2)

puts("After shift by array.shift(2):\n\n")
puts(array.inspect, "\n")
